src/os/bluestore/BlueStore.cc

```diff
@@ -4555,6 +4555,13 @@ int BlueStore::_open_db(bool create)
 	bdev->get_size() * (cct->_conf->bluestore_bluefs_min_ratio +
 			    cct->_conf->bluestore_bluefs_gift_ratio);
       initial = MAX(initial, cct->_conf->bluestore_bluefs_min);
+      if (cct->_conf->bluefs_alloc_size % min_alloc_size) {
```

Comment:
This is not right. We haven't choose a proper min_alloc_size yet by now. See mkfs.